src/Controller/CRUDController.php

```diff
@@ -135,6 +135,9 @@ public function listAction(Request $request): Response
             $exportFormats = $exporter->getAvailableFormats($this->admin);
         }
 
+        /**
+         * @psalm-suppress DeprecatedMethod
```

Comment:
A NEXT_MAJOR replace with `render()` should be added (?)

Reply:
The `NEXT_MAJOR` comment is pedantic because the annotation should provide enough information

Reply:
When we had to resolve all the 4.x NEXT_MAJOR some of them were not obvious, so there is never too much comment.

`@psalm-suppress` just tell to ignore the deprecation but does not explain how to solve it.

At least
```
NEXT_MAJOR: Remplace this method by `$this->render`
```
above `renderWithExtraParams` definition might be more useful than `remove this method`.